Cargo.toml

```diff
@@ -114,6 +114,8 @@ anyhow = "1.0"
 assert-json-diff = "2.0"
 async-trait = "0.1"
 atty = "0.2"
+aws-config = "1.5.6"
+aws-sdk-kms = "1.44"
```

Comment:
If this works for X.Y, we might want to drop the Z

Reply:
Oh, missed that one thanks!

Reply:
[23d2040](https://github.com/FuelLabs/sway/pull/6578/commits/23d2040861f2336b076bed001828850674da00e3)